common/src/Assets/Texture.cpp

```diff
@@ -97,6 +97,22 @@ namespace TrenchBroom {
             return m_name;
         }
 
+        const IO::Path& Texture::absolutePath() const {
```

Comment:
This is now unused, right? Why do we still want to store it?

Reply:
Useful for debugging, also we might want to display it in a tooltip.